channel.yaml

```diff
@@ -0,0 +1,35 @@
+# Example channels config
+channels:
+- name: stable
+  latest: v1.17.4+k3s1
+- name: latest
+  latestRegexp: .*
+  excludeRegexp: "-"
+- name: testing
+  latestRegexp: .*
+- name: v1.16
+  latestRegexp: v1\.16\..*
+  excludeRegexp: "-"
```

Comment:
Is it better to put `^[^+]+-`.  I think that regexp is correct.  Basically it says ignore strings with `-` where the characters before `-` are anything but `+`.  That way v1.16.3+some-thing will not match but v1.16.3-rc.1 will.  Regex's are hard...